meson.build

```diff
@@ -202,9 +202,10 @@ endif
 
 cne_conf.set('HAS_UINTR_SUPPORT', false)
 if cc.has_argument('-muintr') and cc.get_id() == 'gcc' and cc.has_header('x86gprintrin.h')
-    message('GCC Version: ' + cc.version())
-    cne_conf.set('HAS_UINTR_SUPPORT', true)
-    add_project_arguments('-muintr', language: 'c')
+    if cc.links('#include <x86gprintrin.h>\nint main(){_stui();return 0;}', args: '-muintr')
```

Comment:
The has_argument and has_header checks are redundant, if you are going to check them again inside the links check.

I'd just use
```meson
if cc.get_id() == 'gcc' and cc.links(..., name: '-muintr')
```